openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/MetadataToESConfigForm.tsx

```diff
@@ -25,109 +24,87 @@ interface Props {
 const { Text } = Typography;
 
 export default function MetadataToESConfigForm({
-  metadataToESConfig,
   handleMetadataToESConfig,
   handlePrev,
   handleNext,
 }: Props) {
   const { t } = useTranslation();
-  const [caCerts, SetCaCerts] = useState<string>('');
-  const [regionName, SetRegionName] = useState<string>('');
-  const [timeout, SetTimeout] = useState<number>(0);
-  const [useAwsCredentials, SetUseAwsCredentials] = useState<boolean>(false);
-  const [useSSL, SetUseSSL] = useState<boolean>(false);
-  const [verifyCerts, SetVerifyCerts] = useState<boolean>(false);
 
-  const handleCaCertsChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    SetCaCerts(event.target.value);
+  const getFieldLabels = (label: string, description: string) => {
+    return (
+      <Space direction="vertical">
+        <Text>{label}</Text>
+        <Text className="input-field-descriptions">{description}</Text>
+      </Space>
+    );
```

Comment:
Make it a component instead of a function.

Reply:
👍